### PR TITLE
#4555 Rename Reference.ZOOKEEPER_CLUSTER to Reference.ZOOKEEPER, Refe…

### PR DESCRIPTION
**Pull request by chia7712**
…rence.BROKER_CLUSTER to Reference.BROKER and Reference.WORKER_CLUSTER to Reference.WORKER

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4555)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
javadoc error is traced by #5168

File: ohara-manager/client/src/api/apiInterface/definitionInterface.ts [hunk 1]
```diff
@@ -23,9 +23,9 @@ export enum Necessary {
 export enum Reference {
   NONE = 'NONE',
   TOPIC = 'TOPIC',
-  ZOOKEEPER_CLUSTER = 'ZOOKEEPER_CLUSTER',
-  BROKER_CLUSTER = 'BROKER_CLUSTER',
-  WORKER_CLUSTER = 'WORKER_CLUSTER',
+  ZOOKEEPER = 'ZOOKEEPER',
```
[the remark this answers is not in the document]
Comment (saivirtue) [reply ohara-manager/client/src/api/apiInterface/definitionInterface.ts:26]: Let me check this.

[the remark this answers is not in the document]
Comment (saivirtue) [reply ohara-manager/client/src/api/apiInterface/definitionInterface.ts:26]: Looks all the relative code has been changed, thanks 😃

**Comment by jackyoh** [comment]
很像 javadoc 的註解格式有誤

**Comment by jackyoh** [comment]
> 很像 javadoc 的註解格式有誤

這應該跟 #5167  有關

**Pull request merged**